app/main.py

```diff
@@ -1 +1,33 @@
-# write your code here
+class Animal:
+    alive = []
+
+    def __init__(self,
+                 name: str,
+                 health: int = 100,
+                 hidden: bool = False) -> None:
+        self.name = name
+        self.health = health
+        self.hidden = hidden
+        Animal.alive.append(self)
+
+    def __repr__(self) -> str:
+        return "{Name: " + f"{self.name}" + ", Health: " + \
+               f"{str(self.health)}" + ", Hidden: " + \
+               f"{str(self.hidden)}" + "}"
```

Comment:
use interpolation here

Reply:
Send me please interpolation example

Reply:
<img width="579" alt="image" src="https://user-images.githubusercontent.com/72562232/194534063-5146cdab-736c-462c-aa09-0b5385b3fa8e.png">

Reply:
Okay. How can I add {Name: correctly to f string?

Reply:
f"{{Name: "